core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleParser.java

```diff
@@ -1318,4 +1319,9 @@ protected void throwEOFException() throws RDFParseException {
 	protected int getLineNumber() {
 		return lineNumber;
 	}
+
+	public StringBuilder getBuilder() {
```

Comment:
This should be a protected method so it isn't exposed to users, just to subclasses as necessary.